src/composables/useDisabledJoinPool.ts

```diff
@@ -35,10 +35,10 @@ function doesRequireAllowListing(pool: Pool, account: string): boolean {
 
 export function useDisabledJoinPool(pool: Pool) {
   const { account } = useWeb3();
-  const { vettedTokenList } = useTokenLists();
+  const { allowlistedTokenList } = useTokenLists();
```

Comment:
Maybe we could just name this `balancerTokenlist` instead? That's what it's called in the tokenlist repo.

Reply:
That makes sense